tests/workflow.spec.ts

```diff
@@ -51,4 +49,59 @@ test.describe(`Testfälle für den Test von workflows: Umgebung: ${process.env.U
       await Startseite.text_h2_Ueberschrift.click();
     })
   })  
+
+  test('SPSH-215 Passwort Reset', async ({ page}) => {
+    const Landing = new LandingPage(page);
+    const Login = new LoginPage(page);
+    const Startseite = new StartPage(page);
+    const UserManagement = new UserManagementPage(page);
+    const UserManagementDetail = new UserManagementDetailPage(page);
+    const Header = new HeaderPage(page);
+    const username_lastname = 'Max';
+    let new_password = '';
+
+    await test.step(`Portal öffnen ${URL_PORTAL}`, async () => {
+      await page.goto(URL_PORTAL);
+    })
+
+    await test.step(`Annmelden mit Administrator ${USER}`, async () => {
+      await Landing.button_Anmelden.click();
+      await Login.login(USER, PW); 
```

Comment:
Ist es sinnvoll, die Konstante USER vorausschauend in ADMIN umzubenennen? Ich stelle mir vor, dass der Workflow zukünftig auch User vs. Admin Tests beinhalten soll.

Reply:
Passt